wagtail/contrib/redirects/management/commands/import_redirects.py

```diff
@@ -0,0 +1,182 @@
+import os
+
+import tablib
+from django.core.management.base import BaseCommand
+from tablib.formats import registry
+from wagtail.contrib.redirects.forms import RedirectForm
+from wagtail.core.models import Site
+
+
+class Command(BaseCommand):
+    help = "Imports redirects from .csv, .xls, .xlsx"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--src", help="Path to file", type=str, required=True,
+        )
+        parser.add_argument(
+            "--site_id", help="The site where redirects will be associated", type=int,
+        )
+        parser.add_argument(
+            "--permanent",
+            help="Save redirects as permanent redirects",
+            type=bool,
+            default=True,
+        )
+        parser.add_argument(
+            "--from_index",
+            help="The column where to read from link",
+            default=0,
+            type=int,
+        )
+        parser.add_argument(
+            "--to_index", help="The column where to read to link", default=1, type=int,
+        )
+        parser.add_argument(
+            "--dry_run",
+            default=False,
+            help="Run only in test mode, will not create redirects",
+            type=bool,
+        )
+        parser.add_argument(
+            "--ask", help="Ask before creating", default=False, type=bool,
```

Comment:
Same here, this could be an optional arg, meaning you can type `--ask` instead of `--ask TRUE`

https://docs.djangoproject.com/en/3.0/howto/custom-management-commands/#accepting-optional-arguments